examples/functional_tests.py

```diff
@@ -0,0 +1,94 @@
+import sys
+from openrazer.client import DeviceManager
+from openrazer.client import constants as razer_constants
+
+def main():
+    # Create a DeviceManager. This is used to get specific devices
+    device_manager = DeviceManager()
+
+
+    print("Found {} Razer devices".format(len(device_manager.devices)))
+    print()
+
+
+    prompt = 'noprompt' not in sys.argv
```

Comment:
I like to use `argparse` to handle passed in parameters. It also helps users understand how to use the tool. This is no a blocker though.